internal/fs/fs.go

```diff
@@ -244,12 +244,20 @@ func createFileCacheHandler(cfg *ServerConfig) (fileCacheHandler *file.CacheHand
 	// gcsfuse related files.
 	cacheLocation = path.Join(cacheLocation, util.FileCache)
 
+	//Panic in case cacheLocation does not have required permissions
+	cacheLocationErr := util.CreateCacheDirectoryIfNotPresentAt(cacheLocation)
+	if cacheLocationErr != nil {
+		panic(fmt.Sprintf("error while creating file cache directory: %v", cacheLocationErr.Error()))
```

Comment:
can we start the error with "createFileCacheHandler: " ?

Reply:
why not? done